apps/www/content/docs/installation.mdx

```diff
@@ -55,9 +60,16 @@ const { fontFamily } = require("tailwindcss/defaultTheme")
 
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  darkMode: ["class", '[data-theme="dark"]'],
-  content: ["app/**/*.{ts,tsx}", "components/**/*.{ts,tsx}"],
+  darkMode: ["class"],
+  content: ["pages/**/*.{ts,tsx}", "components/**/*.{ts,tsx}"],
```

Comment:
I use `app` here for Next.js 13 app dir. I'm wondering if we should leave Tailwind CSS config to the official doc.

Reply:
Or leave a comment:
```js
  content: [
    "./app/**/*.{js,ts,jsx,tsx}", // Note the addition of the `app` directory for Next.js 13 (experimental app dir)
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ],
 ```